ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java

```diff
@@ -8229,9 +8299,17 @@ private void handleLineage(LoadTableDesc ltd, Operator output)
       Path tlocation = null;
       String tName = Utilities.getDbTableName(tableDesc.getDbTableName())[1];
       try {
+        String suffix = "";
+        if (AcidUtils.isTransactionalTable(destinationTable)) {
```

Comment:
use AcidUtils.isTableSoftDeleteEnabled() it checks SOFT_DELETE_TABLE property as well

Reply:
Updated.